mbzirc_seed/config/team.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- model_name: 'usv'
+  model_type: 'usv'
+  position:
+    xyz: [-1462, -16.5, 0.3]
+    rpy: [0, 0, 0]
```

Comment:
add an arm here? since that's pretty much needed in the second part of the challenge

Reply:
Done.